reaper.go

```diff
@@ -72,6 +71,11 @@ func NewReaper(ctx context.Context, sessionID string, provider ReaperProvider, r
 		WaitingFor: wait.ForListeningPort(listeningPort),
 	}
 
+	// include reaper-specific labels to the reaper container
+	for k, v := range reaper.Labels() {
```

Comment:
We identified that the Labels() function contains more information: about the SessionID

https://github.com/testcontainers/testcontainers-go/pull/570/files#diff-e8ab4ec17e978db949cd15726ba0efa9c89fd714e7e6e0aa537ed7fc1bbde7fdR150-R155